sources/packages/backend/libs/sims-db/src/entities/student-appeal.model.ts

```diff
@@ -71,3 +71,8 @@ export class StudentAppeal extends RecordDataModel {
   )
   studentAssessment?: StudentAssessment;
 }
+
+export enum AppealType {
```

Comment:
This enum should be at the service level since it is not related to the DB models.

Reply:
The file `apps/api/src/utilities/datatable-config.ts` is in the service layer, which reinforces the point of not adding the `AppealType` in the DB layer.

Reply:
Explaining further, the enum can be moved inside apps\api\src\services\student-appeal\student-appeal.model.ts.

First, the enum is used only by the API; hence, there is no need to have it declared outside the API.
Second, the entity model is part of the DB/Repository layer; if it is not DB-related or required for a DB model, it should not be at that level.
So, the right place to have an enum that is used only by the API controller and its services is in the services of the API.
Please let me know if it needs further clarification.